nucypher/policy/conditions/lingo.py

```diff
@@ -150,7 +151,7 @@ def __process(self, *args, **kwargs):
                 operator = task
                 yield operator
             else:
-                raise RuntimeError
+                raise TypeError
```

Comment:
For debugging purposes - although feels like we would fail before we get here
```suggestion
                raise TypeError(f"Unrecognized type {type(task)} for ConditionLingo")
```